.pullapprove.yml

```diff
@@ -0,0 +1,56 @@
+# You *must* specify "version: 2"
+version: 2
+
+requirements:
+  signed_off_by:
+    required: false
+
+always_pending:  # A pull request matching any of these conditions will always have a "pending" status for this group.
+  title_regex: '(WIP|wip)'
+  explanation: 'Work in progress...'
+
+# Group settings to apply to all groups by default, optionally being overridden later
+group_defaults:
+  reset_on_push: # Whether or not approval statuses get reset to "pending" when new commits are pushed to a PR
+    enabled: false # Set to false for efficiency
+  reset_on_reopened: # Whether or not approval statuses get reset to "pending" when a pull request is re-opened
+    enabled: true
+  author_approval: # Whether or not the author must approve the pull request.
+    required: false
+    ignored: true
+  conditions:
+    branches:
+      - master  # Only require reviews when PR is merging into master
+    labels:
+      exclude:
+        - skip-review  # don't need to review if it has the "skip-review" label, designed for "fire-fighting" PRs
+    files:
+      include:
+        - '*.md'
+      exclude: # don't need to review if the PR only changes these files
+        - 'README.md'
+  approve_by_comment:
+    enabled: true
+    approve_regex: '(^[Aa]pproved)|(^LGTM)'  # overrides the default '^:\+1:'
+  reject_value: -1 # Essentially allows rejections to cancel out approvals
+
+# Groups of reviewers and their respective settings
+groups:
+  computational-biologists:
+    required: 1 # require approval from at least one computational biologist
+    users:
+      - ambrosejcarr
+      - jishuxu
+      - mckinsel
+      - timothytickle
```

Comment:
Although @kbergin and @TimothyTickle have lots of expertise to offer, given their roles on the team, they have not typically been giving official approvals on PRs and I don't expect them to in future. I think we should remove them from the lists of required reviewers. This wouldn't stop them from reviewing PRs but means we would still require at least one engineer and one comp bio from the team to review, in addition to them. Again, we could revisit in future if it turns out there are PRs where it makes sense for them to be the official approvers.